pandas/tests/tools/test_to_datetime.py

```diff
@@ -1014,6 +1015,36 @@ def test_convert_object_to_datetime_with_cache(
         )
         tm.assert_series_equal(result_series, expected_series)
 
+    @pytest.mark.parametrize("cache", [True, False])
+    @pytest.mark.parametrize(
+        "constructor",
+        (
+            partial(Series, dtype="object"),
+            list,
+            np.array,
+        ),
+    )
+    @pytest.mark.parametrize(
+        "datetimelikes,expected_values",
+        (
+            (
+                (datetime(3000, 1, 1, 0, 0, 0, 0, pytz.UTC),) * (start_caching_at + 1),
+                (NaT,) * (start_caching_at + 1),
```

Comment:
i think the multiplication can be done inside the test?

Reply:
Done